examples/slice/main.go

```diff
@@ -0,0 +1,20 @@
+package main
+
+import (
+	"fmt"
+	"github.com/bitfield/script"
+	"log"
+)
+
+func main() {
+	// Get a Slice with all the running process PID
+	pids, err := script.Exec("ps -ea").Column(1).Slice()
```

Comment:
Yes! This is a great use case for Slice(), exactly what we need.

However, wherever possible I try to avoid calling userland tools like `ps` in example programs. After all, one of the big applications of the `script` library is in scratch containers and other places where we can't rely on having any of the standard Unix tools installed. 

Suppose we did something like in the introductory README example:

```go
script.Args().Concat().Match("Error").First(10).Stdout()
```

Only instead of sending the error matches to `Stdout()`, we range over them as a slice and print them out:

```go
for _, e := range errors {
	fmt.Printf("ERROR: %s", e)
}
```

Reply:
Yes you are right this is a way better example.